src/main/java/org/broadinstitute/consent/http/db/DataSetPropertiesMapper.java

```diff
@@ -25,22 +25,27 @@ public DataSetDTO map(int index, ResultSet r, StatementContext ctx) throws SQLEx
         String consentId = r.getString("consentId");
         Integer alias = r.getInt("alias");
         if (!dataSets.containsKey(dataSetId)) {
-            dataSetDTO = new DataSetDTO( new ArrayList<>());
+            dataSetDTO = new DataSetDTO(new ArrayList<>());
             dataSetDTO.setConsentId(consentId);
             dataSetDTO.setAlias(alias);
             dataSetDTO.setDataSetId(dataSetId);
             dataSetDTO.setActive(r.getBoolean("active"));
-            dataSetDTO.setTranslatedUseRestriction((r.getString("translatedUseRestriction") == null) ? null : r.getString("translatedUseRestriction"));
-            DataSetPropertyDTO property = new DataSetPropertyDTO("Dataset Name",r.getString("name"));
-            dataSetDTO.getProperties().add(property);
-            property = new DataSetPropertyDTO(r.getString(PROPERTY_KEY),r.getString(PROPERTY_PROPERTYVALUE));
+            dataSetDTO.setTranslatedUseRestriction(r.getString("translatedUseRestriction"));
+            DataSetPropertyDTO property = new DataSetPropertyDTO("Dataset Name", r.getString("name"));
             dataSetDTO.getProperties().add(property);
```

Comment:
In http/models/dto/DataSetDTO.java, create an addProperties() function and call that here, instead of calling a list from a getter and modifying it.

Reply:
Nice, thank you! Will fix that up for the final version.